code/modules/projectiles/ammo_datums.dm

```diff
@@ -1143,6 +1168,113 @@ datum/ammo/bullet/revolver/tp44
 /datum/ammo/bullet/railgun/on_hit_mob(mob/M, obj/projectile/P)
 	staggerstun(M, P, weaken = 1, stagger = 3, slowdown = 2, knockback = 3, shake = 0)
 
+/datum/ammo/tx54
+	name = "20mm airburst grenade"
+	icon_state = "grenade"
+	hud_state = "grenade_airburst"
+	hud_state_empty = "grenade_empty"
+	handful_icon_state = "20mm_airburst"
+	handful_amount = 3
+	ping = null //no bounce off.
+	sound_bounce	= "rocket_bounce"
+	flags_ammo_behavior = AMMO_EXPLOSIVE|AMMO_ROCKET|AMMO_SUNDERING
+	armor_type = "bomb"
+	damage_falloff = 0
+	shell_speed = 2
+	accurate_range = 12
+	max_range = 15
+	damage = 12			//impact damage from a grenade to the dome
+	penetration = 0
+	sundering = 0
+	shrapnel_chance = 0
+	bullet_color = LIGHT_COLOR_FIRE
+	bonus_projectiles_type = /datum/ammo/bullet/tx54_spread
+	bonus_projectiles_scatter = 30
+
+/datum/ammo/tx54/on_hit_mob(mob/M, obj/projectile/proj)
+	staggerstun(M, proj, stagger = 0, slowdown = 0.5, knockback = 1, shake = 0)
+	var/main_proj_angle = Get_Angle(proj.firer, M)
```

Comment:
only used once after this, dont need to define a new var here

Reply:
What should I do instead?